draft-sharabayko-mops-srt.md

```diff
@@ -2862,13 +2862,21 @@ IV = (MSB(112, Salt) << 2) XOR (PktSeqNo)
 
 # Security Considerations
 
-SRT supports confidentiality of user data using stream ciphering based on AES.
-Session keys for ciphering are delivered through control packets during handshake,
-with the protection by Key Encryption Key, which is generated by a sender and receiver
-with pre-shared secret such as passphrase. As in UDT, careful uses of SYN Cookies
-may help to deter denial of service attacks. Appropriate security policy including
-key size, key refresh period, as well as passphrase should be managed by security
-officers, which is out of scope of the present document.
+SRT provides confidentiality of user data using stream ciphers as
+specified in {{encryption}}, which is the only improvement of
+security upon UDT. That means other limitations from UDT remain
+as they are, and SRT needs to consider any security protection
+mechanisms at an upper layer depending on its security requirements.
+Integrity protection of user data can also be supported as described
+in counter mode of {{SP800-38A}}, and its usage details are out of scope for the present document.
```

Comment:
SRT provides confidentiality of user data using stream ciphers as specified in 
{{encryption}}. This is the only improvement over UDT, which does not have any 
specific security mechanisms (SRT therefore needs to consider security 
requirements at an upper layer).

Reply:
Please don't mention UDT here. UDT is not some official protocol.

Reply:
A draft proposal

```suggestion
SRT provides confidentiality of the payload using stream ciphers and a pre-shared private key as
specified in {{encryption}}. The security can be compromised if the pre-shared passphrase
is known to the attacker.

On the protocol control level, SRT does not encrypt packet headers. Therefore it has some vulnerabilities similar to TCP {{RFC6528}}:

- a peer tells a counterpart his public IPs during the handshake, which is visible to any attacker,

- an attacker may count the number of systems behind a Network Address Translator (NAT) by establishing a number of SRT connections and identifying the number of different sequence number "spaces", depending on the implementation of the initial sequence number generation.

- eavesdropper can also hijack existing connections only if it can steal the IP and port of one of the parties. If some stream addresses an existing SRT receiver by its SRT socket ID, IP and port number, but arrives from a different IP or port, then SRT receiver ignores it.

- SRT has certain protection from DoS attacks, see {{handshake-messages}}.

```

Reply:
Sounds good to me. What's **eavesdropper** in the 3rd point? Or it's a misprint?

Reply:
"eavesdropper" is someone listening to others secretly.